apps/andi/lib/andi/input_schemas/dataset.ex

```diff
@@ -0,0 +1,334 @@
+defmodule Andi.InputSchemas.Technical do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Andi.InputSchemas.DatasetSchemaValidator
+
+  @no_dashes_regex ~r/^[^\-]+$/
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+  schema "technical" do
+    field(:dataName, :string)
+    field(:orgName, :string)
+    field(:private, :boolean)
+    has_many(:schema, Andi.InputSchemas.DataDictionary)
+    field(:sourceFormat, :string)
+    has_many(:sourceHeaders, Andi.InputSchemas.SourceHeaders)
+    has_many(:sourceQueryParams, Andi.InputSchemas.SourceQueryParams)
+    field(:sourceType, :string)
+    field(:systemName, :string)
+    field(:sourceUrl, :string)
+    field(:topLevelSelector, :string)
+    field(:cadence, :string)
+    field(:orgId, :string)
+
+    belongs_to(:dataset, Andi.InputSchemas.Dataset, type: :string, primary_key: true, foreign_key: :dataset_id)
+  end
+
+  def changeset(technical, %SmartCity.Dataset.Technical{} = changes) do
```

Comment:
we'll break these out and also move the conversion related stuff back into InputConverter